tests/drivers/build_all/video/mimxrt1170_evk_mimxrt1176_cm7.overlay

```diff
@@ -8,6 +8,8 @@
  * (and be extended to test) real hardware.
  */
 
+#include <zephyr/dt-bindings/video/video-interfaces.h>
```

Comment:
Test changes need to be with the change to the binding to preserve bisectability